src/Syntax.jl

```diff
@@ -373,7 +373,7 @@ None. This mutates the given dyvars vector.
 function parse_dyvar!(dyvars::Vector{Tuple{Symbol,Expr}}, dyvar::Expr)
     push!(dyvars, parse_dyvar(dyvar))
  end
- 
```

Comment:
Emacs is set to remove all trailing whitespace. Sorry for the noise in this PR! 😬